jupyter_foundation.md

```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
```

Comment:
FYI, we still need to link to a draft of the charter (which we are close to having a draft of, and will be available for review before we vote). Also, we should clarify here that the charter is the source of truth for this information, and any discrepancy between this document and the charter should defer to the charter.

Reply:
Ana and I pushed a few changes, including adding a sentence around this.